src/main.rs

```diff
@@ -391,16 +405,21 @@ fn run() -> Result<()> {
                     )
                 }
             };
+            let profile = match (release, profile) {
+                (true, Some(_)) => bail!("cannot set both `--release` and `--profile`"),
```

Comment:
I think this can be done with [Arg::conflicts_with](https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with)?